spec/features/consumer/shopping/shopping_spec.rb

```diff
@@ -536,4 +537,10 @@ def shows_products_without_customer_warning
     expect(page).to have_no_content "This shop is for customers only."
     expect(page).to have_content product.name
   end
+
+  def wait_for_debounce
+    # The auto-submit on these specific form elements (add to cart) now has a small built-in
+    # waiting period before submitting the data...
+    sleep 0.6
```

Comment:
Capybara has better built-in mechanisms :see_no_evil: I suffered a bit with hidden sleeps in a test suite in the past.

Reply:
In this case the expectation will totally break using Capybara's built in expectations...